zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/handlers/ListViewProcessInstanceFromIncidentHandlerTest.java

```diff
@@ -1,209 +0,0 @@
-            .setTreePath("PI_111/FN_3/FNI_456677/PI_23432");
-
-    final BatchRequest mockRequest = mock(BatchRequest.class);
-
-    final Map<String, Object> expectedUpdateFields = new LinkedHashMap<>();
-    expectedUpdateFields.put(TREE_PATH, inputEntity.getTreePath());
-    // when
-    underTest.flush(inputEntity, mockRequest);
-    // then
-    verify(mockRequest, times(1))
-        .upsert(indexName, inputEntity.getId(), inputEntity, expectedUpdateFields);
-  }
-
-  @Test
-  public void shouldUpdateEntityFromRecordWithCallActivity() {
-    // given
-    final long processDefinitionKey1 = 999L;
-    final Long pi1Key = 111L;
-    final Integer callActivityIndex1 = 3;
-    final String callActivityId1 = "callActivity1";
-    final long callActivity1Key = 123L;
-    final long processDefinitionKey2 = 888L;
-    final Long pi2Key = 222L;
-    final Integer callActivityIndex2 = 1;
-    final String callActivityId2 = "callActivity2";
-    final long callActivity2Key = 234L;
-    final Long pi3Key = 333L;
-    final long flowNodeInstanceKey = 345L;
-    final Record<IncidentRecordValue> incidentRecord =
-        createIncidentRecord(
-            List.of(
-                List.of(pi1Key, callActivity1Key),
-                List.of(pi2Key, callActivity2Key),
-                List.of(pi3Key, flowNodeInstanceKey)),
-            List.of(callActivityIndex1, callActivityIndex2),
-            List.of(processDefinitionKey1, processDefinitionKey2, 777L));
-    processCache.put(
-        processDefinitionKey1,
-        new CachedProcessEntity(null, null, List.of("0", "1", "2", callActivityId1)));
-
-    processCache.put(
-        processDefinitionKey2, new CachedProcessEntity(null, null, List.of("0", callActivityId2)));
-
-    // when parent process instance
-    final ProcessInstanceForListViewEntity processInstanceForListViewEntity1 =
-        new ProcessInstanceForListViewEntity().setId(String.valueOf(pi1Key));
-    underTest.updateEntity(incidentRecord, processInstanceForListViewEntity1);
-    // then
-    assertThat(processInstanceForListViewEntity1.getTreePath()).isEqualTo("PI_111");
-
-    // when called process 2nd level
-    final ProcessInstanceForListViewEntity processInstanceForListViewEntity2 =
-        new ProcessInstanceForListViewEntity().setId(String.valueOf(pi2Key));
-    underTest.updateEntity(incidentRecord, processInstanceForListViewEntity2);
-    // then
-    assertThat(processInstanceForListViewEntity2.getTreePath())
-        .isEqualTo("PI_111/FN_callActivity1/FNI_123/PI_222");
-
-    // when called process 3rd level
-    final ProcessInstanceForListViewEntity processInstanceForListViewEntity3 =
-        new ProcessInstanceForListViewEntity().setId(String.valueOf(pi3Key));
-    underTest.updateEntity(incidentRecord, processInstanceForListViewEntity3);
-    // then
-    assertThat(processInstanceForListViewEntity3.getTreePath())
-        .isEqualTo("PI_111/FN_callActivity1/FNI_123/PI_222/FN_callActivity2/FNI_234/PI_333");
```

Comment:
Svetlana is working on getting this test part of the other handler UT.

This we should definitely something we should cover.

🔧 I think we should have some test with more nesting inside the same process instance.

Reply:
> I think we should have some test with more nesting inside the same process instance.

You mean more scopes? Additional scopes like subprocesses won't go into inter tree path. But I added them to record data: https://github.com/camunda/camunda/pull/25531/commits/65f102fce9e8abca643e0c5308be0e7e40dfa0e5#diff-394591f11d1232612dc04e957c52abd3140961414df36bda9546351c809e4ba0R373